py2pack/__init__.py

```diff
@@ -336,10 +341,51 @@ def _normalize_license(data):
 def _prepare_template_env(template_dir):
     # setup jinja2 environment with custom filters
     env = jinja2.Environment(loader=jinja2.FileSystemLoader(template_dir))
+
+    def _rpm_format_requires(req):
+        req[0] = "python-" + req[0]
+        part1 = " ".join(req[0:3])
+        part2 = ""
+        if len(req) > 3:
+            req[3] = "python-" + req[3]
+            part2 = ", " + " ".join(req[3:])
+        return part1 + part2
+
+    def _parenthesize_version(req):
```

Comment:
nitpick: This function does complex require formatting, it could be great to have some comment with examples of what it does, input -> output so it will be easier to understand in the future.